### PR TITLE
Connect

### PR DESCRIPTION
**Pull request by chadwcarlson**
## Description

- Metabase version in flat file, rather than variable (so source ops can work)
-  Adds source op and cron

## Related Issue

Got no issues friend.

### Please drop a link to the issue here:

See above

## Motivation and Context

- @hacktivist123 we want to open automatic updates on these templates ultimately, and this is one way we can do this.
- Source ops is a paid feature we want to show off

## How Has This Been Tested?
Please describe in detail how you tested your changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc.

## Types of changes
What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following list, and put an `x` in all the boxes that apply. If you're unsure about what any of these mean, don't hesitate to ask. We're here to help!

- [ ] I have read the contribution guide
- [ ] I have created an issue following the issue guide
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hacktivist123** [comment]
@chadwcarlson Didn't know source ops existed, I like it, it LGTM 🚀

File: metabase.version [hunk 1]
```diff
@@ -1 +1 @@
-v0.40.1
```
Comment (hacktivist123) [inline metabase.version]: @chadwcarlson why are you downgrading the version here 🤔

Comment (chadwcarlson) [reply metabase.version]: only to test

Comment (hacktivist123) [reply metabase.version]: okay, noted

**Comment by chadwcarlson** [comment]
@all-contributors please add @hacktivist123 for code

**Pull request merged**